hippod/api_object_post.py

```diff
@@ -290,6 +290,34 @@ def object_index_initial_add_major(app, sha_major, xobj):
     object_index_update(app, object_index)
 
 
+def check_anchor(app, xobj):
+    anchor = False
+    achievements = xobj['achievements']
+    for achievement in achievements:
+        if 'anchor' in achievement:
+            anchor = True
+    return anchor
+
+
+def update_lifetime(app, sha_major, sha_minor, anchor):
+    conf = app['CONF']
+    garbage = hippod.garbage_handler_container_achievements.GHContainerAchievements()
+    ok, subcontainer = hippod.api_shared.read_subcont_obj_by_id(app, sha_major, sha_minor)
+    if not ok:
+        log.error("can not read subcontainer by id '{}/{}'".format(sha_major, sha_minor))
```

Comment:
is it ok to simple process here? What ahppen if not ok, then subcontainer is not defined and later you access elements of subcontainer. So why do just a log.error() and process normally ...?